database/redis/metric.go

```diff
@@ -235,6 +235,16 @@ func (connector *DbConnector) RemovePatternWithMetrics(pattern string) error {
 	return nil
 }
 
+// RemoveMetricRetention remove metric retention.
+func (connector *DbConnector) RemoveMetricRetention(metric string) error {
```

Comment:
seems missed test? :)

Reply:
thanks! Added https://github.com/moira-alert/moira/pull/769/commits/298740a3aebdbcef59d2bd0dec8889e48f43688e#diff-06def16160d138dfc8f942b74c5737e0bca49b562474aca4343b9cde3ab2985fR187, I think positive case (when key exists) is enough

Reply:
yuuhuu!